soccer/gameplay/tactics/line_up.py

```diff
@@ -46,6 +46,13 @@ def execute_running(self):
         for i in range(6):
             pt = self._line.get_pt(0) + (self.diff * float(i))
             self.subbehavior_with_name("robot" + str(i)).pos = pt
+            subbehavior = self.subbehavior_with_name("robot" + str(i))
+         
+
+        for bhvr in self.all_subbehaviors():
+            if (bhvr.robot != None):
+                bhvr.robot.face(robocup.Point(0,0))
```

Comment:
Having them all face the goal exposes the bug we were running into when we were moving the robots at comp. I think it should be left in here until we resolve that since it's pretty serious...